src/settings/MappingProfiles/initialDetails/INSTANCE.js

```diff
@@ -123,33 +123,69 @@ const INSTANCE = {
     }],
   }, {
     name: 'precedingTitles',
-    enabled: true,
+    enabled: false,
     path: 'instance.precedingTitles',
     value: '',
     subfields: [{
       order: 0,
       path: 'instance.precedingTitles[]',
       fields: [{
         // order: 0,
-        name: 'precedingInstanceId',
+        name: 'precedingTitlesTitle',
+        enabled: true,
+        path: 'instance.precedingTitles[].precedingTitlesTitle',
+        value: '',
+      }, {
+        // order: 1,
+        name: 'precedingTitlesHrid',
+        enabled: true,
+        path: 'instance.precedingTitles[].precedingTitlesHrid',
+        value: '',
+      }, {
+        // order: 2,
+        name: 'precedingTitlesIsbn',
+        enabled: true,
+        path: 'instance.precedingTitles[].precedingTitlesIsbn',
+        value: '',
+      }, {
+        // order: 3,
+        name: 'precedingTitlesIssn',
         enabled: true,
-        path: 'instance.precedingTitles[].precedingInstanceId',
+        path: 'instance.precedingTitles[].precedingTitlesIssn',
```

Comment:
Update path and name fields according to the [json schema](https://github.com/folio-org/mod-inventory/blob/master/ramls/instance.json#L403), please. Do the same for the `succeedingTitles` fields.
```
precedingTitlesTitle -> title
precedingTitlesHrid -> hrid
precedingTitlesIsbn -> identifiers[].value
precedingTitlesIssn -> identifiers[].value
```

Reply:
Done